tests/suites/test_suite_ssl.function

```diff
@@ -1651,6 +1697,9 @@ void perform_handshake( handshake_test_options* options )
         TEST_ASSERT( mbedtls_endpoint_init( &server, MBEDTLS_SSL_IS_SERVER,
                                             options->pk_alg, NULL, NULL, NULL ) == 0 );
     }
+
+    mbedtls_ssl_conf_authmode( &server.conf, options->srv_auth_mode );
```

Comment:
Why was this added? What does it have to do with handshake fragmentation testing? Should this be added in its own commit?

Reply:
By default, server does not check client's certificates so I had to set it manually to be required. This forces the client to send more date to the server during handshake. Without that handshake messages from the client would be to small to be fragmented.

Reply:
OK, I see how it is related to fragmentation now. Could you please explain this aspect of your implementation in the git commit message?

Reply:
I added an explanation to the commit message